### PR TITLE
Add more unit tests

### PR DESCRIPTION
**Pull request by nadiam84**
Another addition to the #128

### CODE REVIEW

File: tests/unit/getOppositeVariation.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import chai from 'chai';
+const { expect } = chai;
+import getOppositeVariation from '../../src/popper/utils/getOppositeVariation';
+
+describe('utils/getOppositeVariation', () => {
+  it('should return correct values', () => {
+    expect(getOppositeVariation('start')).to.equal('end');
```
Comment (FezVrasta) [inline tests/unit/getOppositeVariation.js]: maybe test the -end and -start variations as well?

File: tests/unit/getOppositePlacement.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import chai from 'chai';
+const { expect } = chai;
+import getOppositePlacement from '../../src/popper/utils/getOppositePlacement';
+
+describe('utils/getOppositePlacement', () => {
+  it('should return correct values', () => {
+    expect(getOppositePlacement('top')).to.equal('bottom');
+    expect(getOppositePlacement('bottom')).to.equal('top');
+    expect(getOppositePlacement('left')).to.equal('right');
+    expect(getOppositePlacement('right')).to.equal('left');
```
Comment (FezVrasta) [inline tests/unit/getOppositePlacement.js]: maybe test the -end and -start variations as well? top-start should return bottom-start

Comment (nadiam84) [inline tests/unit/getOppositePlacement.js]: done!

**Comment by FezVrasta** [comment]
Looks good, may you please rebase on master and fix the build? Thank you!

**Comment by nadiam84** [comment]
Done!

**Pull request merged**